src/components/new-request/search.vue

```diff
@@ -184,16 +184,6 @@
         <template>
           <!-- Search for business identifier or name if NR request action is one of [CHG, AML, CNV, REH] -->
           <BusinessLookup v-if="isBusinessLookup" />
-
-          <template v-if="businessIdentifier">
-            <dl>
-              <dt class="font-weight-bold mr-2">Business Name:</dt>
-              <dd>{{businessName}}</dd>
```

Comment:
Are businessName and businessIdentifier still used by other code, or can they be deleted?

Reply:
Thanks.